lib/parse.js

```diff
@@ -10,6 +10,12 @@ var DOMParser = require('xmldom').DOMParser;
 
 exports.parse = parse;
 
+var NODE_TYPES = {
```

Comment:
Ok, will do.
Btw: https://docs.travis-ci.com/user/sauce-connect/ and https://docs.travis-ci.com/user/jwt might help with getting the sauce labs tests to run for PRs from forks.